### DISCUSSION TITLE
Security Onion 2 Installation

### DISCUSSION THREAD

**Discussion by MrAnderson28** [Security-Onion-Solutions/securityonion#2231] [Unsupported Versions] [answered]
Hello all. I recently downloaded Security Onion 2 ISO and am trying to set it up in Virtual Box. I have followed the tutorials and allocated the required amount of resources. Once I boot the VM I am presented with the installation screen (image below)

![image](https://user-images.githubusercontent.com/39415382/101258236-d2168000-36e6-11eb-8f33-d7af72f8b69e.png)

However, the next screen that I am presented with an option is this one:

![image](https://user-images.githubusercontent.com/39415382/101258283-1b66cf80-36e7-11eb-84d3-b2d4d03b9064.png)

None of the tutorial videos I have watched show this screen. I am just curious when this says it will destroy all data, is this just referencing the virtual partition you set up for this? Or is it referencing the actual physical partition of your system? Also, is this screen normal to receive with Sec Onion 2 or is this indicative that I set something up incorrectly? Curious since I havent seen this screen elsewhere.

Any advice would be appreciated!

**Comment by sleepingbel** [comment, marked as answer]
Hello,

This means that de install of SO will format the virtual drive for creating the needed partitions for SO.
It will not destroy data outside the VM.

This is a warning for when someone recycles a virtual machine, that already contains data, to install SO on. This is a normal warning you get when installing an OS. 

Regards
Bart

**Comment by defensivedepth** [comment]
@sleepingbel already answered the question, however, our free training on YouTube covers the install step-by-step, in case you run into other issues:

Security Onion Essentials - https://www.youtube.com/watch?v=BFdTHGcsmkk&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=2

**Reply by MrAnderson28** [reply]
Thank you for the video links! I hadnt come across this series of videos yet.

**Comment by MrAnderson28** [comment]
@sleepingbel @defensivedepth Great! Thank you both for your responses on this. I appreciate the assistance!
